src/Data/Char/Properties.agda

```diff
@@ -60,13 +116,31 @@ private
   unit-test = p _
 
 ------------------------------------------------------------------------
--- Structures
-
-setoid : Setoid _ _
-setoid = PropEq.setoid Char
-
-decSetoid : DecSetoid _ _
-decSetoid = PropEq.decSetoid _≟_
-
-strictTotalOrder : StrictTotalOrder _ _ _
-strictTotalOrder = On.strictTotalOrder ℕₚ.<-strictTotalOrder toNat
+-- Properties of _<_
+
+infix 4 _<?_
+_<?_ : Decidable _<_
+_<?_ = On.decidable toNat ℕ._<_ ℕₚ._<?_
+
+≈-<-strictTotalOrder : StrictTotalOrder _ _ _
```

Comment:
I think the convention in Data.Nat.Properties has been to add the equality to the end of the type, i.e. `<-strictTotalOrder-≈`. Same with `String` below.